### PR TITLE
Build on jdk Set release target to 8 for jdk11 compiler

### PR DESCRIPTION
**Pull request by pacomendes**
Test running ci build on openjdk11 with release 8

### CODE REVIEW

**Comment by asdf2014** [comment]
ping @Randgalt @leventov

File: .travis.yml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 language: java
 jdk:
-  - oraclejdk8
+  - openjdk11
```
Comment (leventov) [inline .travis.yml]: Can test on both?

Comment (asdf2014) [reply .travis.yml]: @pacomendes Would you please replace it with 

```yml
jdk:
  - openjdk8
  - openjdk11
```

Comment (pacomendes) [reply .travis.yml]: This will likely fail on openjdk8 as the -release option is not recognised.

Comment (asdf2014) [reply .travis.yml]: @pacomendes Raising a new PR to solve this problem https://github.com/apache/curator/pull/321

**Comment by martin-g** [comment]
This PR is obsolete and can be closed!
The project uses Github Actions now!

**Comment by tisonkun** [comment]
We already build for JDK8 and JDK11 on master.

**Pull request closed without merging**